src/frontend/data/bootstrapStore.ts

```diff
@@ -1,24 +1,53 @@
 import { applyMiddleware, compose, createStore } from 'redux';
 import createSagaMiddleware from 'redux-saga';
 
+import { requestStatus } from '../types/api';
 import { AppData } from '../types/AppData';
 import { modelName } from '../types/models';
 import { initialState } from './context/reducer';
 import { rootReducer } from './rootReducer';
 import { rootSaga } from './rootSaga';
+import { TimedTextTracksState } from './timedtexttracks/reducer';
 
 const sagaMiddleware = createSagaMiddleware();
 
 export const bootstrapStore = (appData: AppData) => {
   const composeEnhancers =
     (window as any).__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
 
+  let transcriptsState: TimedTextTracksState | null = null;
```

Comment:
Maybe this should be called `timedTextTracksState` as it's not just transcripts?

Reply:
Yes sorry my mind is still focused on the transcript PR...